feature: add model_version_policy to model config

laurenyu opened the pull request:
*Issue #, if available:*
#121 

*Description of changes:*
This change adds the [model version policy](https://www.tensorflow.org/tfx/serving/serving_config#serving_a_specific_version_of_a_model) to the generated model config.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

ChuyangDeng commented:
Did you tested this change with the pre/post-processing to make sure the python service will parse the inference with versions correctly?

laurenyu commented:
good callout - I added tests for pre-/post-processing and MME. I also deleted some duplicated code - let me know if I was missing something when I thought I could do that 😅
